jupyter_notebook/README.md

```diff
@@ -0,0 +1,81 @@
+# Jupyter Notebook
```

Comment:
It would be nice to have a summary up front explaining what this support allows you to do (e.g., train models through FlexFlow in a Jupyter notebook?) and what the limitations are (e.g., cannot run over multiple machines)